README_docker.md

```diff
@@ -0,0 +1,164 @@
+# Batteries-included drand on Docker
+
+The main readme is [here](./README.md). This README is somewhat not aligned with the Docker setup already proposed (I'm refering to the `Dockerfile` in the repo), *but* it is batteries-included: follow this guide (and only this one) to get a simple, resilient `drand` setup in Docker. In contrast, the docker setup in the repo is not built so the keys remains after recreating the image, etc.
+
+## Prerequisites for this guide
+
+a VPS with the following software setup:
+
+1. `docker >= 17.12`
+2. `docker-compose >= 1.18`
+
+(but typically versions are not important)
+
+on your host:
+
+1. `go >= 1.12`
+
+## Generate keys
+
+Before creating the docker, let's generate keys locally (on the host)
+
+	go get -u github.com/dedis/drand
+	drand generate-keypair <address>
+
+Where `<address>` is for instance `drand.yourserver.com:1234` or `yourserver.com:1234`.
+
+This generates keys in `~/.drand/keys/` on the host.
+
+## Docker & Docker-compose setup
+
+Create a `Dockerfile` with the following contents:
+
+	FROM golang:1.12.5-alpine # update this number when possible, but keep "alpine" for a lightweight image
+	RUN apk update
+	RUN apk upgrade
+	RUN apk add git
+	RUN go get -u github.com/dedis/drand
+	WORKDIR /
+	ENTRYPOINT ["drand", "--verbose", "2", "start", "--listen", "0.0.0.0:8080"]
+
+**Note:** if you intend to use `drand` behind a reverse-proxy, add `--tls-disable` here; please read "Docker behind reverse proxy setup" at the end of this guide. 
+
+Let's continue with docker-compose. It will help keeping our container running, plus manage volumes, etc.
+
+Create the following `docker-compose.yml` :
+
+	drand:
+	  build: .
+	  volumes:
+	    - ./data:/root/.drand
+	  ports:
+	    - "0.0.0.0:1234:8080"
+
+Finally, also create a `data` folder to hold the keys and settings. It __HAS__ to have permissions `740`.
+
+	mkdir data
+	chmod 740 data
+
+Drand can then be built into an image with
+
+	docker-compose up --build
+
+Finally, the image is ready for DKG. Just start it and let it run in background with 
+
+	docker-compose up -d
+
+(one can also set a restart policy in the docker-compose.yml)
+
+## DKG
+
+If you did the setup above, you have a container running, loaded with your keys. It still misses two things:
+
+1. the group.toml file corresponding to other participants. For this, you have to exchange keys manually, e.g., via email.
+2. running the DKG protocol to bootstrap drand.
+
+Fortunately, with our docker-compose volumes, it's now very easy to add things into the running container. Just add your `group.toml` into the root of the `data` folder (__NOT__ in the `data/groups/` folder; this one is manually managed by drand, don't touch it).
```

Comment:
same here for the bold thing. I'd prefer to just have "bold" and not the "_" thing before. Sorry for the nitpicking lol